examples/hosts/app-integration/external-data/src/view/taskListView.tsx

```diff
@@ -129,6 +129,20 @@ export const TaskListView: React.FC<ITaskListViewProps> = (props: ITaskListViewP
 	const { taskList } = props;
 
 	const [tasks, setTasks] = useState<ITask[]>(taskList.getDraftTasks());
+	const [lastSaved, setLastSaved] = useState<number | undefined>();
+	const [failedUpdate, setFailedUpdate] = useState(false);
+	const handleSaveChanges = async (): Promise<void> => {
```

Comment:
Typically you'd want to wrap this callback in a [`useCallback`](https://beta.reactjs.org/reference/react/useCallback) hook to avoid superfluous redeclarations and rerenders.